addons/mrp/models/mrp_workorder.py

```diff
@@ -556,46 +556,39 @@ def _plan_workorder(self, replan=False):
     @api.model
     def gantt_unavailability(self, start_date, end_date, scale, group_bys=None, rows=None):
         """Get unavailabilities data to display in the Gantt view."""
-
```

Comment:
I think the feature did work "by chance": if workcenter_id is not found in the group_bys, the records in a given row can be associated with different workcenters and I don't see the point to mix the unvavailabilities in that case.